packages/flutter/lib/src/material/toggleable.dart

```diff
@@ -46,7 +46,8 @@ abstract class RenderToggleable extends RenderConstrainedBox implements Semantic
       value: _value ? 1.0 : 0.0
     );
     _position = new CurvedAnimation(
-      parent: _positionController
```

Comment:
I think it's better to be explicit in this case.  We'll probably want to use `@required` once that works too.